basics/imports-and-modules.md

```diff
@@ -0,0 +1,57 @@
+# Import ve Modül Sistemi
+
+{{#img-right}}turtle.svg{{/img-right}}
+
+D dilinin ana tasarım amacı tutarlı olmak ve uç durumlardan uzak durmaktır.
+Bu durum [_turtles all the way down_](https://en.wikipedia.org/wiki/Turtles_all_the_way_down) olarak söylenir.
+Bu durumun iyi bir örneği ise `import` ifadesidir. 
+
+## Import
+
+D dilindeki basit bir merhaba dünya uygulaması için, `import` gereklidir.
+`import` ifadesi verilen `modül`deki herkese açık tüm (public) işlev (function) ve türleri
+kullanılabilir/erişilebilir kılar.
+
+### The turtles start falling down
+
+Bir `import` ifadesi kaynak kodun en üst satırında bulunmak zorunda __değil__. 
+Yerel bir şekilde işlevlerin ya da kapsamların içerisinde kullanılabilir.
+Sonraki bölümlerde bunun D dilindeki neredeyse bütün kavramlarda uygulandığını göreceksiniz.
+D dili nedensizce kısıtlamalar uygulamaz.
+
+### Seçici Import
```

Comment:
Seçici Import yerine ise benim önerim şunlar:
"Seçerek Kütüphane Eklemek" yada "Parsiyel Kütüphane Ekleme"
[Parsiyel nedir?](http://www.tdk.gov.tr/index.php?option=com_bts&arama=kelime&guid=TDK.GTS.5b59d0f1298674.52222023)

Reply:
Parsiyel, partial kelimesinden geliyor. Mümkün olduğunca Türkçeleştirebilsek çok güzel olurdu. Ben de çok kararsız kaldım.
"Seçerek Kütüphane Eklemek" olabilir veya "Seçerek Erişim" diye de çevirebiliriz.
Bu durumda import kelimesini erişim diye çevirmiş olacağız ne kadar doğru olur bilmiyorum .

Reply:
Bence anahtar kelimeleri mutlaka korumalıyız (import gibi). Ancak Türkçe okuyan bir kişinin bu kelimenin arkasındaki anlamı kavramasını sağlamalıyız. Yani kelimeyi açıklamalı ama orjinalinide okuyucuya sunmalıyız. 

"Seçerek Erişim" yerine "Seçerek Eklemek" bana daha doğal geliyor. Neticede yapılan iş erişmek değil mevcut kodun içine dahil etmek diye düşünüyorum.

Doğruluk yanlışlık konusuna bence çok takılma seninde mesajında belirttiğin gibi bir şeyler olması (hatalı da olsa) hiç olmamasından daha iyidir. Mevcut olan geliştirilir ancak hiç olmamasından daha iyidir. Aslında Hüseyin'de katılsa iyi olurdu.

Reply:
Aslında erişim dememin sebebi şu, sembolleri erişilebilir kılıyor yoksa o modüle onları eklemiyor. Derleyicinin o dosya içerisinde sembolü bulmasını sağlıyor. 

Hüseyin baya meşgul diye biliyorum dahil olacağını düşünmüyorum doğrusu.

O halde "Seçerek Eklemek" olarak düzeltiyorum. Sonrasında tekrardan değişiklik yapılabilir.

Reply:
Açıklamasını yapınca "Erişim" kelimeside anlamlı geliyor. Büyük ihtimalle böyle bir çok durumla karşılacağız. Bence sadece basit düşünelim. Detaya girip kafamız karışmasın.

"Modülden Seçerek Eklemek" bence gayet güzel görünüyor. Bence tamam.